### DISCUSSION TITLE
Can Epic Ver XCom2 use AML？

### DISCUSSION THREAD

**Discussion by Zzl5970** [X2CommunityCore/xcom2-launcher#328] [General]
You know,some player use Epic to play this game.
I tried once and in that case,this app ask me to open the steam.
Any way to skip the steam check?

**Comment by RevZero** [comment]
There is a way to use AML without Steam, but Epic is not supported due to other incompatibilities.

**Reply by Zzl5970** [reply]
fine, thanks for your answer.
